src/sensors.py

```diff
@@ -0,0 +1,93 @@
+#import BoardInterface
+
+from enum import Enum
+
+# Sensor: Abstract class representing an individual sensor on the test stand.
```

Comment:
Class comments should be in docstrings - e.g.
"""
Sensor: Abstract...
...
...
"""

And they should be immediately beneath the class. In this file for instance, that would be line 10.